dapp/src/utils/numbers.ts

```diff
@@ -218,3 +219,11 @@ export const addLeadingZero = (num: number): string =>
 
 export const getPercentValue = (value: number, maxValue: number) =>
   (value * 100) / maxValue
+
+export const getNumberWithSign = (number: number) =>
+  new Intl.NumberFormat("en-US", {
+    signDisplay: "exceptZero",
+  }).format(number)
+
+export const bigIntStringToNumber = (bigIntString: string) =>
+  Number(BigInt(bigIntString))
```

Comment:
Are we sure is it safe? Why can't we use bigint?

Reply:
https://github.com/thesis/acre/pull/704/commits/b34af63cce0c7ce2ae4b8eca9274744fb5c1febf